### ISSUE TITLE
TypeScript definitions

### ISSUE THREAD

**Issue by nerfpops** [ngryman/reading-time#16]
```
declare module "reading-time" {
    interface ReadingTimeStats {
        text: string
        time: number
        words: number
        minutes: number
    }

    function readingTime(text: string): ReadingTimeStats

    namespace readingTime { }

    export = readingTime
}
```

**Comment by Axel92Dev** [comment]
Hey, thanks a lot for this type definitions.

What about exporting the inner interfaces sot hey can be easily referenced?

```typescript
declare module "reading-time" {
    export interface ReadingTimeOptions {
        wordsPerMinute?: number
        wordBound?: string
    }

    export interface ReadingTimeStats {
        text: string
        time: number
        words: number
        minutes: number
    }

    function readingTime(text: string, options?: ReadingTimeOptions): ReadingTimeStats

    namespace readingTime { }

    export = readingTime
}
```

So the importing project can do something like

```typescript
import { IReadTimeResults } from 'reading-time';

type Props = {
  title: string;
  date: string;
  readingTime: IReadTimeResults;
};
```

Does it look good to you?

Thanks a lot

**Comment by deadcoder0904** [comment]
@Axel92Dev I was looking to do the same thing.

@ngryman Can you release the above thing? Just need to copy-paste the above code in TypeScript definitions :)

**Comment by deadcoder0904** [comment]
@ngryman Just sent a PR, if you can merge & release :)

Till then, I'll use my fork 👍

**Comment by ngryman** [comment]
Merged and released as `1.4.0.`.

**Comment by deadcoder0904** [comment]
Thank you @ngryman that was quick 🚀
